TicTacToe.cabal

```diff
@@ -40,6 +34,10 @@ library
       bmp,
       gloss
   default-language: Haskell2010
+  -- We have to do this because our gloss/GLUT dependencies don't automatically
+  -- propagate this C dependency for some reason
+  pkgconfig-depends:
+        freeglut
```

Comment:
We could probably do something to make `gloss` do this for us. But this works fine for our purposes of getting the native dependency in scope.

Reply:
So this makes Windows work, but it breaks Linux lol. I'm working on a fix :)

Reply:
Ok I just branch in the cabal file based on OS now. It works on Linux and Wine for me.